pkg/plugins/golang/v3/init.go

```diff
@@ -36,7 +36,7 @@ import (
 // Variables and function to check Go version requirements.
 var (
 	goVerMin = golang.MustParse("go1.17")
-	goVerMax = golang.MustParse("go1.18")
+	goVerMax = golang.MustParse("go2.0alpha1")
```

Comment:
Should I revert this change, since the e2e tests were failing because of the issue in the tests repo.

Reply:
Reverted